### DISCUSSION TITLE
Is it possible to remove the padding for Lspsaga hover_doc?

### DISCUSSION THREAD

**Discussion by cj** [nvimdev/lspsaga.nvim#1475] [Q&A]
Hi!

Is it possible to remove the padding for `Lspsaga hover_doc`?

![image](https://github.com/user-attachments/assets/2daa9dcd-b520-4442-813c-7fdeadfbc7f5)

**Comment by glepnir** [comment]
currently neovim not support this. i will implement this in neovim core later.
